README.md

```diff
@@ -99,11 +99,16 @@ In this project, I have built a simple HTML list of To Do tasks. This simple web
 Install this project with:
 
 - Open your command prompt or terminal and run.
--  git clone https://github.com/RKanyoro/AwesomeBooks-ES6.git
+- git clone https://github.com/RKanyoro/To-DoList.git
+- cd To-DoList
+- git checkout interactiveList (For Contributions/Comments on Pull Request 4)
+- npm install webpack webpack-cli --save-dev (Win OS).
+- npm start (To Launch App).
+
 
```

Comment:
-  [ ] 🙏Please fix the live demo link as it is broken now. If you have not deployed your project, you can simply remove🗑️ the entire live demo section. This will make it easier for someone interested to try your project and see the live demo.
![unknown_001](https://github.com/RKanyoro/To-DoList/assets/63445083/65e7670c-79b9-4404-88ff-2945768d0eb2)

- [ ] 🙏Please fix the link for the issue page as it is not pointing to the correct location. In this case, you are not allowed to remove🗑️  this section, so you should update it with the correct issue page link. This will make it easier for others to create issues in your project.
![image](https://github.com/RKanyoro/To-DoList/assets/63445083/e1d8eab8-7471-4234-bb39-4c471ead7817)